e2e/tests/plugins/plot/logPlot.e2e.spec.js

```diff
@@ -48,15 +48,15 @@ test.describe('Log plot tests', () => {
 
         // FIXME: Get rid of the waitForTimeout() and lint warning exception.
         // eslint-disable-next-line playwright/no-wait-for-timeout
-        await page.waitForTimeout(1 * 1000);
+        // await page.waitForTimeout(1 * 1000);
 
         // refresh page and wait for charts and ticks to load
-        await page.reload({ waitUntil: 'networkidle'});
-        await page.waitForSelector('.gl-plot-chart-area');
-        await page.waitForSelector('.gl-plot-y-tick-label');
+        // await page.reload({ waitUntil: 'networkidle'});
+        // await page.waitForSelector('.gl-plot-chart-area');
+        // await page.waitForSelector('.gl-plot-y-tick-label');
```

Comment:
remove commented out code

Reply:
I'll need to leave these into place to fix in the future